rollup.bzl

```diff
@@ -0,0 +1,12 @@
+load("//lib:rollup.bzl", "rollup_bundle")
+def rollup(bundles):
```

Comment:
I don't think the extra indirection to this file is useful, just inline this into the BUILD.bazel ?

Reply:
Sure, you are right, I have removed all separated `rollup.bzl` and move the definition into `BUILD.bazel`.